libs/block-editor/src/lib/plugins/bubble-link-form.plugin.ts

```diff
@@ -0,0 +1,233 @@
+import { Editor, posToDOMRect, isNodeSelection } from '@tiptap/core';
+import { EditorState, Plugin, PluginKey, Transaction } from 'prosemirror-state';
+import { EditorView } from 'prosemirror-view';
+import tippy, { Instance, Props } from 'tippy.js';
+
+import { ComponentRef } from '@angular/core';
+import { BubbleMenuLinkFormComponent } from '../extensions/components/bubble-menu-link-form/bubble-menu-link-form.component';
+
+// Interface
+import { PluginStorage } from '../extensions/bubble-link-form.extension';
+
+interface PluginState {
+    toggle: boolean;
+}
+
+export interface BubbleLinkFormProps {
+    pluginKey: PluginKey;
+    editor: Editor;
+    element: HTMLElement;
+    tippyOptions?: Partial<Props>;
+    storage: PluginStorage;
+    component?: ComponentRef<BubbleMenuLinkFormComponent>;
+}
+
+export type BubbleLinkFormViewProps = BubbleLinkFormProps & {
+    view: EditorView;
+};
+
+export class bubbleLinkFormView {
```

Comment:
```suggestion
export class BubbleLinkFormView {
```

Classes name should be capital-case.